File: Python/Estructuras de control/Estructuras condicionales/CalculadoraSimple.py
# 2. Escriba el algoritmo para simular una calculadora. Se debe leer los operandos y el
# operador. 

# Entradas
print("Calculadora")
num1 = float(input("Ingrese el primer número: "))
num2 = float(input("Ingrese el segundo número: "))
operador = input("Ingrese el operador: ")

#procesos
if operador == "+":
    resultado = num1 + num2
elif operador == "-":
    resultado = num1 - num2
elif operador == "*":
    resultado = num1 * num2
elif operador == "/":
    resultado = num1 / num2
else:
    resultado = "Operador no válido"

# Salidas
print(f"El resultado de {num1} {operador} {num2} es: {resultado}")

File: C++/Estructuras de control/Estructuras condicionales/CalculadoraSimple.cpp

// 2. Escriba el algoritmo para simular una calculadora. Se debe leer los operandos y el
// operador. 

#include <iostream>
using namespace std;

int main() {
    // Entradas
    cout << "Calculadora" << endl;
    float num1, num2;
    char operador;
    cout << "Ingrese el primer número: ";
    cin >> num1;
    cout << "Ingrese el segundo número: ";
    cin >> num2;
    cout << "Ingrese el operador: ";
    cin >> operador;

    // Procesos
    float resultado;
    if (operador == '+') {
        resultado = num1 + num2;
    } else if (operador == '-') {
        resultado = num1 - num2;
    } else if (operador == '*') {
        resultado = num1 * num2;
    } else if (operador == '/') {
        resultado = num1 / num2;
    } else {
        cout << "Operador no válido" << endl;
        return 1;
    }

    // Salidas
    cout << "El resultado de " << num1 << " " << operador << " " << num2 << " es: " << resultado << endl;

    return 0;
}